Asteroids game as a separate new feature is created for issue #27. game made using pygame is done in its own file with no dependency on other issues

dbla581 opened the pull request:
Issue #27 is closed after this pull. 
Asteroids fits the retro feel that other games in the repo have. Simplistic vector inspired graphics and 5 different asteroids. Bullets are shot from the the users and asteroids hit split in two.  Option to go back to main menu is implemented as well as score stored in a text file. Stats page will depend on this. 0 changes have been made so far to the main menu or any file outside of the new asteroids.py. Followed structure of snake game. Resource folder is made to hold asteroids images.

ananyaahluwalia01 reviewed (changes requested):
Wow, I love your implementation of this awesome game! I really like how the comments throughout your code highlight why you used certain numbers or took certain routes to fix problems. I also really love the graphics you have created for the game, and your use of classes.

A possible suggestion for improvement in readability could be if you switched the numbers used in the code to be constants that are initialised at the start - for example, the numbers "151" and "150" initialised as a constant could be easier to understand than the magic numbers they appear to be now. Also, there could be changes to the code to better adhere to python naming conventions (like snake_case for variables instead of camelCase, and starting class names off with a capital letter).

Other than that the game looks really good to me!

dbla581 commented:
Thanks for the suggestions!  I'm not used to python so they were very helpful.  I've implemented those changes and removed all code smells that sonarcloud detected. The code is now easier to understand and change. Expanding the game by adding larger asteroids can now be done by updating the "max_size_asteroid" and "number_asteroids" variables, rather than dealing with magic numbers.
